wifiphisher/common/interfaces.py

```diff
@@ -160,6 +181,8 @@ def __init__(self, name, card_obj, mac_address):
         self._has_ap_mode = False
         self._has_monitor_mode = False
         self._is_wireless = False
+        self._is_managed_by_nm = False
```

Comment:
sorry what I want to prevent is the recovering mac process:
Since if we recover the mac address of the internet card, the interface will down and cause the internet access is unavaliable.

```python
for interface in self._active:
            adapter = self._name_to_object[interface]
            if not adapter.is_internet_access:
                mac_address = adapter.original_mac_address

                self.set_interface_mac(interface, mac_address)
```